src/universal/containers/GitHubReposMenuRoot/GitHubReposMenuRoot.js

```diff
@@ -45,36 +50,40 @@ const GitHubReposMenuRoot = ({atmosphere, projectId, setError, clearError, close
       variables={{teamId, service: GITHUB}}
       subscriptions={subscriptions}
       render={({error, props}) => {
-        // TODO handle the error within the menu
-        if (error) {
-          return <ErrorComponent height={'14rem'} error={error} />;
-        }
-        if (props) {
-          return (<GitHubRepoListMenu
-            viewer={props.viewer}
-            teamId={teamId}
-            projectId={projectId}
-            closePortal={closePortal}
-            setError={setError}
-            clearError={clearError}
-            setLoading={setLoading}
-          />);
-        }
-        return <SetLoading setLoading={setLoading} />;
+        return (
+          <TransitionGroup appear style={{overflow: 'hidden'}}>
```

Comment:
nesting conditionals is bleh. doing it in jSX is double bleh.
if this ends up being the best pattern, I'd like to abstract it out into another component so i just pass in props, errors & then let that component handle the ugly JSX once instead of ugly JSX for every Root.